.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,27 @@
+name: reviewdog
+
+on: [push, pull_request]
+
+jobs:
+  misspell:
+    name: runner / misspell
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v3
+      - uses: haya14busa/action-cond@v1
+        id: reporter
+        with:
+          cond: ${{ github.event_name == 'pull_request' }}
+          if_true: "github-pr-review"
+          if_false: "github-check"
+      - name: misspell
+        uses: reviewdog/action-misspell@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: ${{ steps.reporter.outputs.value }}
+          pattern: |
+            *.go
```

Comment:
We don't have to exlude file types

Reply:
`pattern` is target.
Currently target only go files.

If you need request other types, Please comment it.